docs/source/pages/payload.rst

```diff
@@ -227,3 +227,76 @@ If you decide to use an RSA or an EC algorithm, then you **must** provide Sanic
         public_key=public_ec_key,
         private_key=private_ec_key,
         algorithm='ES256')
+
+
++++++++++++++++++
+Encrypted Payload
++++++++++++++++++
+
+Sometimes you may want to hide payload content. So it should be useful if you can encrypt this content. :samp:`sanic-jwt-payload-encrypt` plugin provides encryption for the Payload. To install plugin only do:
```

Comment:
Slight change...

```
Sometimes you may want to hide payload content, and it would be useful if you could encrypt this content. :samp:`sanic-jwt-payload-encrypt` plugin provides encryption for the payload. To install it:
```